### ISSUE TITLE
Update analyzer to automatically build binaries using github actions

### ISSUE THREAD

**Issue by timreyes** [pierremolinaro/canfd-plugin-for-saleae-logic-analyzer#2]
Hi! I'm a Support Engineer at Saleae and we’ve made improvements to our analyzer build process. Specifically, we cleaned up the build system, and switched to cmake, so there aren't multiple project files anymore. By implementing the changes outlined [here](https://github.com/saleae/SampleAnalyzer/#updating-an-existing-analyzer-to-use-cmake--github-actions) you will be able to provide the community automatic builds in addition to a simplified build system. Let us know if you think you can make the changes or if you’d like us to make the changes for you.

Additionally, if you would like to update your analyzer so that results appear in Logic 2’s data table you can follow the steps outlined [here](https://support.saleae.com/saleae-api-and-sdk/protocol-analyzer-sdk/framev2-hla-support-analyzer-sdk). Note that if you upgrade your analyzer to FrameV2 it will not be backward compatible with Logic 1.

We really appreciate your time and effort! Please let us know if you have any feedback or questions. Thanks!

**Comment by pierremolinaro** [comment]
Hello,

I didn't realize that the SDK had changed, I will update the analyzer, as well as the one for CAN 2.0B.

Thanks for the links.  I will also use FrameV2.

Pierre

**Comment by pierremolinaro** [comment]
Hi, it is done.

I have updated the CANFD analyzer, added the workflows/build.yml, and made a release. I think all is ok, I have checked that an analyzer.zip is available with the release, and it contains the dynamic libraries for Windows, Linux and Mac.

I have also updated my CAN 2.0B analyzer i n the same way (https://github.com/pierremolinaro/can20B-plugin-for-saleae-logic-analyzer <https://github.com/pierremolinaro/can20B-plugin-for-saleae-logic-analyzer>), can you add it to the Community Shared Low Level Analyzers list ?

Using Logic2 is very pleasant, I really appreciate it. The only drawback is the simulation mode enabling method (we have discussed about that one year ago). If your are interested, I have written a little launch agent for Mac that permanently enables simulation. It is described in the readme.md of both aalyzers, feel free for using it if you are interested.

Best regards,

Pierre

**Comment by timreyes** [comment]
@pierremolinaro Thanks so much! I'll go ahead and close this Issue now.

I've also added your CAN 2.0B analyzer to our list of community analyzers below:
https://support.saleae.com/community/community-shared-protocols 

You're right about making simulation mode enabled by default. Sorry for the hassle with that. I've added that to the backlog to discuss with the team. I personally haven't had any issues with the feature, so from my point of view, it should be safe to get included by default in upcoming relases.
